pkg/detectors/brandfetch/v2/brandfetch_test.go

```diff
@@ -0,0 +1,111 @@
+package brandfetch
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/engine/ahocorasick"
+)
+
+func TestBrandFetch_Pattern(t *testing.T) {
```

Comment:
Could you please align the pattern test cases with the updates I recently made for the A and B detectors?
Thanks in advance!